src/gdf_table.cuh

```diff
@@ -160,7 +160,8 @@ __global__
 void gather_valid_mask( gdf_valid_type const * const input_mask,
                         gdf_valid_type * const output_mask,
                         index_type const * const __restrict__ gather_map,
-                        index_type const num_rows)
+                        index_type const num_rows,
+                        index_type const input_mask_length)
```

Comment:
Make sure to update the Doxygen documentation about what is `num_rows` vs `input_mask_length`.